.buildkite/pipeline.yml

```diff
@@ -74,6 +74,21 @@ steps:
       - github_commit_status:
           context: "Lint Translations"
 
+  #################
+  # Lint Ruby Tooling
+  #################
+  #
+  # TODO: This ought to be run on a Docker or Ubuntu agent as there's nothing that requires macOS.
+  # It's currently in the stock setup because... it was faster to build a proof of concept for it this way;
+  # I don't know how to pass the `DANGER_GITHUB_API_TOKEN` secret that we need to a Docker agent.
+  #
+  # This step uses Danger to run RuboCop, but it's "agnostic" about it.
+  # That is, it outwardly only mentions RuboCop, not Danger
```

Comment:
It also occurred to me that we could use our Peril instance to do all this.

But, as far as I could see, there is no Peril plugin for RuboCop. That's not to say we couldn't get it to do it... but this seems more straightforward for the moment.

Reply:
It'd be great to get rid of peril altogether, so this IMHO is a step in the right direction 🙂